app/pods/components/bottom-menu/template.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="bottom-menu flex-grid white-border-top">
+  {{#link-to '2016.home' classNames="small-text padding--50"}}2016 Website{{/link-to}}
+  {{#link-to 'seventeen' classNames="small-text padding--50"}}Site Credits{{/link-to}}
+  {{#link-to 'seventeen' classNames="small-text padding--50"}}Twitter{{/link-to}}
+  {{#link-to 'seventeen' classNames="small-text padding--50"}}Tickets{{/link-to}}
+  {{#link-to 'seventeen' classNames="small-text padding--50"}}2016 Talks{{/link-to}}
+  {{#link-to 'seventeen' classNames="small-text padding--50"}}Contact{{/link-to}}
```

Comment:
Need to update a few of these links. Can change to A tags with same classes for any external.